rbac/migration_tool/sharedSystemRolesReplicatedRoleBindings.py

```diff
@@ -0,0 +1,645 @@
+"""
+Copyright 2019 Red Hat, Inc.
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU Affero General Public License as
+published by the Free Software Foundation, either version 3 of the
+License, or (at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU Affero General Public License for more details.
+
+You should have received a copy of the GNU Affero General Public License
+along with this program.  If not, see <https://www.gnu.org/licenses/>.
+"""
+
+import json
+from typing import Callable, FrozenSet, Type
+
+from migration_tool.ingest import add_element, cleanNameForV2SchemaCompatibility
+from migration_tool.migrator import (
+    V1group,
+    V1permission,
+    V1resourcedef,
+    V1role,
+    V2boundresource,
+    V2group,
+    V2role,
+    V2rolebinding,
+)
+
+
+Permissiongroupings = dict[V1resourcedef, list[str]]
+Perm_bound_resources = dict[str, list[V2boundresource]]
+
+group_perms_for_rolebinding_fn = Type[
+    Callable[
+        [str, Permissiongroupings, Perm_bound_resources, FrozenSet[V1group]],
+        FrozenSet[V2rolebinding],
+    ]
+]
+
+system_roles = {}
+
+
+def add_system_role(role: V2role):
+    """Add a system role to the system role map."""
+    system_roles[frozenset(role.permissions)] = role
+
+
+# Cost management system roles (do not currently exist, authored to represent cost
+# management permissions: https://github.com/project-koku/koku/blob/main/koku/koku/rbac.py#L25 )
+
+add_system_role(V2role("cost_administrator", frozenset(["cost_management_all_all"])))
```

Comment:
@coderbydesign should not we generate those ones from config ? 
Maybe to develop some mechanism/task as pre-step for migration ?

@merlante Did you use script for it ?

Reply:
I see. Yea, better to pre-generate it and load this when migrating. Updating now